src/core/libmaven/mzSample.cpp

```diff
@@ -495,6 +495,11 @@ void mzSample::parseMzMLSpectrumList(const xml_node &spectrumList)
 		if (string2float(precursorMzStr) > 0)
 			precursorMz = string2float(precursorMzStr);
 
+		string precursorIsolationStr = isolationWindow["isolation window lower offset"];
+		float precursorIsolationWindow = 1.0;
```

Comment:
Just like earlier, let us start declaring float with the proper suffix (i.e., `float precursorIsolationWindow = 1.0f;`).